app/src/main/res/drawable/ic_pause_circle_filled_white_24dp.xml

```diff
@@ -2,6 +2,7 @@
   android:width="24dp"
   android:height="24dp"
   android:tint="#FFFFFF"
+  android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Reply:
Done,